.eslintrc

```diff
@@ -25,221 +32,152 @@
     }
   },
   "rules": {
-    "react/jsx-no-undef": 0,
+    "react-hooks/exhaustive-deps": "error",
+    "react/jsx-boolean-value": ["error", "never", { "always": [] }],
+    "react/jsx-closing-bracket-location": ["error", "line-aligned"],
+    "react/jsx-closing-tag-location": "error",
+    "react/jsx-curly-spacing": ["error", "never", { "allowMultiline": true }],
+    "react/no-arrow-function-lifecycle": "error",
+    "react/no-invalid-html-attribute": "error",
+    "react/jsx-no-useless-fragment": "error",
+    "react/jsx-no-constructed-context-values": "error",
+    "react/jsx-fragments": ["error", "syntax"],
+    "react/jsx-no-duplicate-props": ["error", { "ignoreCase": true }],
+    "react/jsx-pascal-case": [
+      "error",
+      {
+        "allowAllCaps": true,
+        "ignore": []
+      }
+    ],
+    "react/no-danger": "warn",
+    "react/no-did-update-set-state": "error",
+    "react/no-will-update-set-state": "error",
+    "react/self-closing-comp": "error",
+    "react/jsx-no-undef": ["error", { "allowGlobals": true }],
 
     /*Possible Errors */
-    "no-cond-assign": [1, "except-parens"],
-    "no-console": 0,
-    "no-constant-condition": 1,
-    "no-control-regex": 1,
-    "no-debugger": 1,
-    "no-dupe-args": 1,
-    "no-dupe-keys": 1,
-    "no-duplicate-case": 0,
-    "no-empty-character-class": 1,
-    "no-empty": 0,
-    "no-ex-assign": 1,
-    "no-extra-boolean-cast": 1,
-    "no-extra-parens": 0,
-    "no-extra-semi": 1,
-    "no-func-assign": 1,
+    "no-console": "off",
     "no-inner-declarations": [1, "functions"],
-    "no-invalid-regexp": 1,
-    "no-irregular-whitespace": 1,
-    "no-negated-in-lhs": 1,
-    "no-obj-calls": 1,
-    "no-regex-spaces": 1,
-    "no-reserved-keys": 0,
-    "no-sparse-arrays": 1,
-    "no-unexpected-multiline": 1,
-    "no-unreachable": 1,
-    "use-isnan": 1,
-    "valid-jsdoc": 0,
-    "valid-typeof": 1,
 
     /* Best Practices */
-    "accessor-pairs": 0,
-    "block-scoped-var": 0, // see Babel section
-    "complexity": 0,
-    "consistent-return": 0,
-    "curly": 0,
-    "default-case": 0,
-    "dot-notation": [
-      0,
-      {
-        "allowKeywords": true,
-        "allowPattern": ""
-      }
-    ],
-    "dot-location": [1, "property"],
-    "eqeqeq": 1,
```

Comment:
eqeqeq won't actually be activated by default :(

Reply:
Thanks for picking this up! I've added it back in